packages/runtime/src/helpers/edge.ts

```diff
@@ -378,6 +378,11 @@ export const writeEdgeFunctions = async ({
   const nextConfig = nextConfigFile.config
   const usesAppDir = nextConfig.experimental?.appDir
 
+  await copy(getEdgeTemplatePath('../vendor'), join(edgeFunctionRoot, 'vendor'))
+  netlifyConfig.functions['*'] = {
+    deno_import_map: join(INTERNAL_EDGE_FUNCTIONS_SRC, 'vendor', 'import_map.json'),
```

Comment:
This gives a TypeScript error as `deno_import_map` doesn't exist in `FunctionsObject`. Do we need to update the type to support import maps?

Reply:
yup, good catch! doing that in https://github.com/netlify/build/pull/5305